pkg/divert/istio/divert.go

```diff
@@ -91,8 +93,12 @@ func (d *Driver) Destroy(ctx context.Context) error {
 	return err
 }
 
-func (d *Driver) GetDivertNamespace() string {
-	return ""
+func (d *Driver) UpdatePod(pod apiv1.PodSpec) apiv1.PodSpec {
+	return pod
```

Comment:
Just to clarify for future revisit of the PR. We have to update the pod only when the weaver implementation and update the vs in the istio implementation

Reply:
Yes, that's right!